storage-resize-images/functions/src/index.ts

```diff
@@ -59,6 +59,8 @@ export const generateResizedImage = functions.storage.object().onFinalize(
   async (object): Promise<void> => {
     logs.start();
     const { contentType } = object; // This is the image MIME type
+    const absolutePathList = config.absolutePathList.split(","); // Convert list to an array
+    const tmpFilePath = path.dirname(object.name);
```

Comment:
`dirname` does not return a leading `/` if the provided path doesn't start with one. The regex for `ABSOLUTE_PATH_LIST` requires one, therefore this will never match. There needs to be a leading `/`.